examples/player/simid_player.js

```diff
@@ -282,13 +288,16 @@ class SimidPlayer {
    * @return {!Object}
    */
   getNonlinearDimensions_() {
-    let newDimensions = {};
-    newDimensions.x = document.getElementById('x_val').value;
-    newDimensions.y = document.getElementById('y_val').value;
-    newDimensions.width = document.getElementById('width').value;
-    newDimensions.height = document.getElementById('height').value;
-
-    return newDimensions;
+    if(this.resizeDimensions_) {
```

Comment:
nit - imo,  if this.resizeDimensions reads a little weird here

Reply:
What do you think would sound better here?

Reply:
if you rename the variable to nonLinearDimensions_ it might help with readability here and also at line 407/8

Reply:
hrm, yeah that gave my brain an idea. Why not just call it creativeDimensions_

When you initialize a nonlinear ad set creative dimensions to whatever is in the GUI.  maybe use a function getCreativeDimensionsFromUi().

Then anywhere else, just use this.creativeDimensions_

Would that read easier? Let me know what you think.

Reply:
If we call it creativeDimensions_ do you think it would get confused with the linear creative dimensions? I might prefer nonLinearDimensions_ because it makes it clear the type of creative whose dimensions we want to keep track of.